src/main/java/seedu/address/logic/commands/AddNumberOfRoomsCommand.java

```diff
@@ -14,26 +11,33 @@
 public class AddNumberOfRoomsCommand extends Command {
     public static final String COMMAND_WORD = "addnumberofrooms";
     private int numOfRooms;
-    private String numOfRoomsInString;
 
     /**
      * Creates an AddCommand to add the number of rooms available in a hotel
      */
     public AddNumberOfRoomsCommand(int numOfRooms) {
         this.numOfRooms = numOfRooms;
-        this.numOfRoomsInString = "" + numOfRooms;
     }
     @Override
     public CommandResult execute(Model model) throws CommandException {
-        Path path = model.getPathOfNumberOfRooms();
-        File file = path.toFile();
-        try {
-            FileWriter fileWriter = new FileWriter(file);
-            fileWriter.write(numOfRoomsInString);
-            fileWriter.close();
-        } catch (IOException e) {
-            //Would not happen because the path to numberOfRooms.txt is always present
-        }
+        model.addNumberOfRooms(numOfRooms);
         return new CommandResult(numOfRooms + " rooms are added in a hotel");
```

Comment:
Will be good to include a MESSAGE_SUCCESS attribute or something similar for the string " rooms are added in a hotel" (can refer to the other command classes to see what I mean). 
E.g. in addPatientCommand your have the attribute String MESSAGE_SUCCESS = "New person added: %1$s";
So what you return is return new CommandResult(String.format(MESSAGE_SUCCESS, numPersonToAdd)) instead of the hardcoded string.